configs/components/oasis3mct/oasis3mct.yaml

```diff
@@ -81,7 +82,23 @@ choose_version:
+    #comp_command: 'export ESM_OASIS_DIR=${model_dir}; cd lib; [ -d "yaxt" ] || rm -f yaxt-0.10.0.tar.gz && wget https://swprojects.dkrz.de/redmine/attachments/download/529/yaxt-0.10.0.tar.gz ; [ -d "yaxt" ] || mkdir yaxt && tar -xzvf yaxt-0.10.0.tar.gz -C yaxt --strip-components=1 ; rm yaxt-0.10.0.tar.gz ; cd yaxt; export YAXT_ROOT=$(pwd) ; mkdir -p build; cd build; echo "../configure --prefix=${model_dir}/lib/yaxt --without-regard-for-quality ; make -j 6 install" ; cd ../../ ; [ -d "yac" ] || git clone --branch=release-3.1.0 https://gitlab.dkrz.de/dkrz-sw/yac.git ; cd yac ; export YAC_ROOT=$(pwd) ; mkdir -p build ; cd build ; echo "../configure --enable-lib-core-only --disable-mpi-checks --with-yaxt-root=${model_dir}/lib/yaxt --with-netcdf-root=$NETCDFROOT --prefix=${model_dir}/lib/yac ; make -j 6 install" ; export YAC_INCLUDE="-I${model_dir}/lib/yac/include -I${model_dir}/lib/yaxt/include"; cd ../../../util/make_dir/ ; cp make.ESMTOOLS make.inc ; make -f TopMakefileOasis3 ; cd ../../ ; mv INSTALL_OASIS.ESMTOOLS/lib/libpsmile.MPI1.a INSTALL_OASIS.ESMTOOLS/lib/libpsmile.a ; mkdir -p build/lib/psmile/scrip ; mkdir -p build/lib/psmile/mct/mpeu ; cp INSTALL_OASIS.ESMTOOLS/build-static/lib/psmile.MPI1/*.mod build/lib/psmile/. ; cp INSTALL_OASIS.ESMTOOLS/build-static/lib/scrip/*mod build/lib/psmile/scrip/. ; cp INSTALL_OASIS.ESMTOOLS/build-static/lib/mct/*mod build/lib/psmile/mct/. ; cp INSTALL_OASIS.ESMTOOLS/lib/libmct.a build/lib/psmile/mct/. ; cp INSTALL_OASIS.ESMTOOLS/lib/libmpeu.a build/lib/psmile/mct/. ; cp INSTALL_OASIS.ESMTOOLS/lib/libpsmile.a build/lib/psmile/. ; cp INSTALL_OASIS.ESMTOOLS/lib/libscrip.a build/lib/psmile/scrip/. '
+    install_libs:
+    - INSTALL_OASIS.ESMTOOLS/lib/libpsmile.a
+    - INSTALL_OASIS.ESMTOOLS/lib/libmct.a
+    - INSTALL_OASIS.ESMTOOLS/lib/libmpeu.a
+    - INSTALL_OASIS.ESMTOOLS/lib/libscrip.a
+    #clean_command: 'rm -rf include lib/libpsmile.a lib/libscrip.a lib/libmct.a lib/libmpeu.a'
```

Comment:
```suggestion
```
Is this comment needed still?

Reply:
It can be helpful. Rebuilding only the old parts of OASIS takes about 1min. Rebuilding YAXT+YAC+OASIS takes about 15. If one does developments in OASIS (like I do for stochastic coupling), this different clean command is good to have.